src/components/Movie-list.js

```diff
@@ -0,0 +1,47 @@
+import { Card, CardMedia, CardContent, Typography } from "@mui/material/";
+
+import useMovies from "../API";
+
+const MediaCard = ({ movie }) => {
+  const posterUrl = `https://image.tmdb.org/t/p/w500${movie.poster_path}`;
+  return (
+    <Card sx={{ maxWidth: 345 }} className="Card-style">
+      <CardMedia
+        component="img"
+        sx={{ height: 540 }}
+        title={movie.name}
+        src={posterUrl}
+        alt="pic"
+      />
+      <CardContent>
+        <Typography gutterBottom variant="h5" component="div">
+          {movie.name}
+        </Typography>
+        <Typography variant="body2" color="text.secondary">
+          {movie.description}
```

Comment:
if the description will be too long it brake the card. maybe would be better to slice description to for example to 20 symbols? 
google how to truncate the strings (the same as for arrays)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice